webview/src/experiments/components/table/styles.module.scss

```diff
@@ -463,34 +459,24 @@ th {
       pointer-events: none;
     }
 
-    td:not(.experimentCell):hover::before,
-    td:hover + td::before {
+    .td:not(.experimentCell):hover::before,
+    .td:hover + .td::before {
       background-color: $row-border-selected-color;
     }
   }
 }
 
-.nestedRow {
-  .experimentCell .innerCell {
```

Comment:
Moved these lines to fix a specificity issue but I also think it looks more readable :)